src/parser/jobs/ast/AstWeaving.js

```diff
@@ -0,0 +1,22 @@
+import Weaving from 'parser/core/modules/Weaving'
+import ACTIONS from 'data/ACTIONS'
+
+export default class AstWeaving extends Weaving {
```

Comment:
Malefic III is a 1.5s cast, which I realise I was missing from the core weaving handling. That's been added now, allowing for 1 weave after a 1.5s cast - is this override still required?

Reply:
No, we can remove this override now. I had brought this up the other day, glad we could keep it neat. There's no other reason to check for weaving to my knowledge